client/my-sites/site-settings/form-general.jsx

```diff
@@ -191,6 +192,12 @@ export class SiteSettingsFormGeneral extends Component {
 							},
 						}
 					) }
+					&nbsp;
+					{ site.domain.endsWith( '.wordpress.com' ) && (
```

Comment:
This check feels a bit fragile; is there a better selector for determining if a site is a WordPress.com simple site? I worry there may be edge cases where the URL is not *.wordpress.com but it is a simple site. I'll do a quick investigation.

Reply:
It appears we only show the blogAddress option to non-Jetpack sites, which seems to include sites on `*.wpcomstaging.com` addresses (even if they're on the Free plan). So I think we could remove this check entirely unless there's another edge case I'm missing? LMK

Reply:
I was more conscious about custom domains on a WP.com simple site which I couldn't check - is it still necessary for that usage case? 

I'll update the wording! :)